backend/djangoindia/bg_tasks/send_update.py

```diff
@@ -12,20 +15,36 @@ def send_mass_update_email_task(update_id):
         update = Update.objects.get(id=update_id)
     except Update.DoesNotExist:
         raise ValueError("Update not found")
+    
+    logger.info(f"Starting email task for update ID {update_id} with {update.recipients.count()} recipients.")
+
+    failed_recipients = []
 
-    email_objs = []
     for subscriber in update.recipients.all():
-        email_tuple = (
-            f"Django India: {update.get_formatted_type()}",
-            update.html_template,
-            settings.DEFAULT_FROM_EMAIL,
-            [subscriber.email],
-        )
-        email_objs.append(email_tuple)
-    try:
-        send_mass_mail((email for email in email_objs), fail_silently=False)
-        update.mail_sent = True
-        update.save()
-    except Exception as e:
-        # Didn't throw error at django level even for wrong emails.
-        print(f"Failed to send email : {str(e)}")
+        try:
+            plain_text_content = strip_tags(update.html_template)
+
+            email = EmailMultiAlternatives(
+                subject=f"Django India: {update.get_formatted_type()}",
+                body=plain_text_content,
+                from_email=settings.DEFAULT_FROM_EMAIL,
+                to=[subscriber.email],
+            )
+
+            email.attach_alternative(update.html_template, "text/html")
+
+            email.send(fail_silently=False)
+        
+        except Exception as e:
+            logger.error(f"Failed to send email to {subscriber.email}: {str(e)}")
+            failed_recipients.append(subscriber.email)
```

Comment:
same thing is logged twice. one in the form of list and another single entity, can we reduce this to one log only?